backend/trivia-server.js

```diff
@@ -7,7 +7,7 @@ import { data } from './constants';
 /* @expose */
 export async function getTriviaData() {
   try {
-    return await update('trivia', (result = data) => result);
+    return await update('trivia', () => data);
```

Comment:
This is not exactly the same, as the current code only updates the `trivia` key if it doesn't exist (so allowing you to modify it somehow), and since there's a `template_init_data.json` file in the project, it will have that information initialized.
that said, it's a good behaviour if we want to support upgrades via code updates through changes to `constants`